Doc/library/os.rst

```diff
@@ -1195,6 +1195,32 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: preadv(fd, buffers, offset, flags = None)
+
+   Combines the functionality of :func:`os.readv` and :func:`os.pread`. It
```

Comment:
I noticed you didn't duplicate the definition of `fd` and `buffers` from readv.  I think most of docs explain all the parms, even if they are duplicated from another section.

Reply:
I has hessitant to duplicate because `readv` is the function described inmediately before this one but I suppose that consistency makes sense. Thanks.

Corrected in 1988d1a